src/jit/codegencommon.cpp

```diff
@@ -8644,6 +8644,43 @@ void CodeGen::genFnEpilog(BasicBlock* block)
         unwindStarted = true;
     }
 
+#if !FEATURE_FASTTAILCALL
+    if (jmpEpilog)
+    {
+        GenTree* jmpNode = block->lastNode();
+
+        noway_assert(jmpNode->gtOper == GT_JMP);
+
+        CORINFO_METHOD_HANDLE methHnd = (CORINFO_METHOD_HANDLE)jmpNode->gtVal.gtVal1;
+        CORINFO_CONST_LOOKUP  addrInfo;
+        compiler->info.compCompHnd->getFunctionEntryPoint(methHnd, &addrInfo);
```

Comment:
We will call `getFunctionEntryPoint` again later when `jmpEpilog` is set. Should we remember the result from here?

Reply:
Good idea, fixed